js/data.js

```diff
@@ -453,6 +508,17 @@ var data = {
         dozent: ["Prof. Dr. Patrick Baudisch"],
         nameLV: "HCI Project Seminar Haptics, Personal Fabrication, and Virtual Reality"
     },
+    hciProjectSeminarPISME: {
+        kurz: "HCI Projekt<br />3D-Printing",
+        kennung: [ITSE, HCT, SAMT],
+        lehrform: ["Seminar", "Projekt"],
+        modul: ["IT Systems Engineering", "Vertiefungsgebiete"],
+        semester: ss15,
```

Comment:
Thing is: 'ss15' is a string, and you can iterate over strings in Javascript. Wonder if it might actually interpret that as semesters ['S', 'S', '1', '5'] :).